src/main/resources/db/migrations/V001__CreateSchemaTables.sql

```diff
@@ -0,0 +1,171 @@
+    room VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.court_regions(
+    id UUID PRIMARY KEY NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    region_id UUID REFERENCES regions(id) NOT NULL
+
+);
+
+CREATE TABLE public.courtrooms (
+    id UUID PRIMARY KEY NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    room_id UUID REFERENCES rooms(id) NOT NULL
+);
+
+CREATE TABLE public.cases (
+    id UUID PRIMARY KEY NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL ,
+    case_ref VARCHAR(25) NOT NULL, 
+    test BOOLEAN DEFAULT FALSE,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.participants (
+    id UUID PRIMARY KEY NOT NULL,
+    case_id UUID REFERENCES cases(id) NOT NULL,
+    participant_type PARTICIPANT_TYPE NOT NULL,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE public.bookings (
+    id UUID PRIMARY KEY NOT NULL,
+    case_id UUID REFERENCES cases(id) NOT NULL,
+    date DATE NOT NULL,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.booking_participants (
+    id UUID PRIMARY KEY NOT NULL,
+    participant_id UUID REFERENCES participants(id) NOT NULL,
+    booking_id UUID REFERENCES bookings(id) NOT NULL
+);
+
+CREATE TABLE public.users (
+    id UUID PRIMARY KEY NOT NULL,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    organisation VARCHAR(250),
+    phone VARCHAR(50),
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE public.roles (
+    id UUID PRIMARY KEY NOT NULL,
+    name VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.app_access (
+    id UUID PRIMARY KEY NOT NULL,
+    user_id UUID REFERENCES users(id) NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    role_id UUID REFERENCES roles(id) NOT NULL,
+    last_access TIMESTAMPTZ,
+    active BOOLEAN DEFAULT TRUE,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_on TIMESTAMPTZ DEFAULT NOW(),
+    modified_on TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.capture_sessions (
+    id UUID PRIMARY KEY NOT NULL,
+    booking_id UUID REFERENCES bookings(id) NOT NULL,
+    origin RECORDING_ORIGIN NOT NULL,
+    ingest_address VARCHAR(255),
+    live_output_url VARCHAR(100),
+    started_on TIMESTAMPTZ,
+    started_by_user_id UUID REFERENCES users(id),
+    finished_on TIMESTAMPTZ,
+    finished_by_user_id UUID REFERENCES users(id),
+    status RECORDING_STATUS,
+    deleted BOOLEAN DEFAULT FALSE
+);
+
+CREATE TABLE public.recording_versions (
```

Comment:
It seems like there might be a slight confusion in the terminology here. The table capture_sessions will be used to store information about the initial session where recordings are captured or started and won't be used to store any recordings. Do you think this will pose any confusion?

With regards the parent_recording_id, 100% agree, I think having a direct reference to the parent recording will make querying quicker and will allow for tracing of the original version of the recording unambiguous. 

Good point, raising with Lawrie/Indy !

Reply:
yep agreed :D